data/predefined_classes.txt

```diff
@@ -1,15 +1,6 @@
-dog
-person
-cat
-tv
-car
-meatballs
-marinara sauce
-tomato soup
-chicken noodle soup
-french onion soup
-chicken breast
-ribs
-pulled pork
-hamburger
-cavity
\ No newline at end of file
+stop sign
+yield sign
+speed sign
+traffic light
+deer
+lane marking
```

Comment:
```suggestion
stop sign
sign
speed sign
traffic light
deer
lane marking
person
car
barrier
```

think theres a few added things we need to train on. I suggested to @TankyFranky that we have a generic sign class and a further classifier will do the classificaiton. Keep stop sign and speed sign in their own classes for YOLO though. What do you think?

Reply:
revision looks good, maybe raise at next perception meeting to finalize an initial list for the first stage model? previously tagged images are not automatically updated when the predefined_classes.txt file is updated. namely in the yolo image format `<object-class> <x> <y> <width> <height>`  `object-class` is an integer field corresponding to the label's list position in `predefined-classes.txt` so its especially critical that we have a strong initial list and the file is protected.  I ran into issues before as I was sloppy with my class list file and trained on spoiled data ☠, ill flesh out some more instructions on how to avoid inadvertently adding new labels.